### PR TITLE
provisional text on rel-types for a plan phase in SW life-cycles

### LINKED ISSUE #672: Relationship Types for early software life-cycle phases (in SPDX 2.3)
In order to refer to specific requirement documents/resources and design specifications adding two relationship types could be really useful - especially 

REQUIREMENT_FOR: Is to be used when SPDXRef-A describes, illustrates, or specifies a requirement statement for SPDXRef-B.

SPECIFICATION_FOR: Is to be used when SPDXRef-A describes, illustrates, or defines a design specification for SPDXRef-B.

### PR DESCRIPTION
**Pull request by henkbirkholz**
fixes #672

### CODE REVIEW

File: chapters/file-information.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ The metadata for the file checksum field is shown in Table 39.
 | --------- | ----- |
 | Required | Yes |
 | Cardinality | 1..1 [`SHA1`][SHA-1], others may be optionally provided. |
-| Algorithm | [`SHA1`][SHA-1] is to be used on the file. Other algorithms that can be provided optionally include [`SHA224`][SHA-224], [`SHA256`][SHA-256], [`SHA384`][SHA-384], [`SHA512`][SHA-512], [`SHA3-256`][SHA3-256], [`SHA3-384`][SHA3-384], [`SHA3-512`][SHA3-512], [`BLAKE2b-256`][BLAKE2b-256], [`BLAKE2b-384`][BLAKE2b-384], [`BLAKE2b-512`][BLAKE2b-512], [`BLAKE3`][BLAKE3], [`MD2`][MD2], [`MD4`][MD4], [`MD5`][MD5], [`MD6`][MD6] |
+| Algorithm | [`SHA1`][SHA-1] is to be used on the file. Other algorithms that can be provided optionally include [`SHA224`][SHA-224], [`SHA256`][SHA-256], [`SHA384`][SHA-384], [`SHA512`][SHA-512], [`SHA3-256`][SHA3-256], [`SHA3-384`][SHA3-384], [`SHA3-512`][SHA3-512], [`BLAKE2b-256`][BLAKE2b-256], [`BLAKE2b-384`][BLAKE2b-384], [`BLAKE2b-512`][BLAKE2b-512], [`BLAKE3`][BLAKE3], [`MD2`][MD2], [`MD4`][MD4], [`MD5`][MD5], [`MD6`][MD6], [`ADLER32`][ADLER32] |
```
Comment (henkbirkholz) [inline chapters/file-information.md:175]:
```suggestion
| Algorithm | [`SHA1`][SHA-1] is to be used on the file. Other algorithms that can be provided optionally include [`SHA224`][SHA-224], [`SHA256`][SHA-256], [`SHA384`][SHA-384], [`SHA512`][SHA-512], [`SHA3-256`][SHA3-256], [`SHA3-384`][SHA3-384], [`SHA3-512`][SHA3-512], [`BLAKE2b-256`][BLAKE2b-256], [`BLAKE2b-384`][BLAKE2b-384], [`BLAKE2b-512`][BLAKE2b-512], [`BLAKE3`][BLAKE3], [`MD2`][MD2], [`MD4`][MD4], [`MD5`][MD5], [`MD6`][MD6] |
```

File: chapters/relationships-between-SPDX-elements.md [hunk 2]
```diff
@@ -60,6 +60,8 @@ The relationships between two SPDX elements that are supported are shown in Tabl
 | AMENDS                 | Is to be used when (current) SPDXRef-DOCUMENT amends the SPDX information in SPDXRef-B.               | (Current) SPDX document A version 2 contains a correction to a previous version of the SPDX document A version 1. Note the reserved identifier SPDXRef-DOCUMENT for the current document is required. |
 | PREREQUISITE_FOR       | Is to be used when SPDXRef-A is a prerequisite for SPDXRef-B.                                         | A library `bar.dll` is a prerequisite or dependency for APPLICATION `foo.exe`|
 | HAS_PREREQUISITE       | Is to be used when SPDXRef-A has as a prerequisite SPDXRef-B.                                         | An APPLICATION `foo.exe` has prerequisite or dependency on `bar.dll` |
+| REQUIREMENT_FOR        | Is to be used when SPDXRef-A describes, illustrates, or specifies a requirement statement for SPDXRef-B. | A PDF document that describes a list of disallowed licences to inherit in certain build-subtrees. |
```
Comment (henkbirkholz) [inline chapters/relationships-between-SPDX-elements.md:63]:
```suggestion
| REQUIREMENT_DESCRIPTION_FOR        | Is to be used when SPDXRef-A describes, illustrates, or specifies a requirement statement for SPDXRef-B. | A PDF document that describes a list of disallowed licences to inherit in certain build-subtrees. |
```

**Comment by kestewart** [comment]
@henkbirkholz - could you please add your signoff?

**Comment by henkbirkholz** [comment]
> @henkbirkholz - could you please add your signoff?

Done!

**Comment by tsteenbe** [comment]
Fixed up this PR for @henkbirkholz - did change the commit messages to be more in line with past commits.

**Review by kestewart — APPROVED** [review]
Thanks Henk & Thomas.   Looks good to me.

**Pull request merged**